app/src/main/java/com/kickstarter/libs/utils/AnalyticEventsUtils.kt

```diff
@@ -275,6 +275,7 @@ object AnalyticEventsUtils {
             }
             put("has_add_ons", hasAddOns?.hasAddons() ?: false)
             put("tags", project.tags()?.let { it.joinToString(", ") } ?: "")
+            put("url", project.urls().web().project())
```

Comment:
Should we need to treat these as optionals?

Reply:
I have check model the URL and web models are not optionals

Reply:
okay! Lgtm!